packages/SwingSet/test/zcf-ish-upgrade/test-zcf-ish-upgrade.js

```diff
@@ -26,9 +26,11 @@ test('zcf-ish upgrade', async t => {
     },
   };
 
-  const kernelStorage = initSwingStore().kernelStorage;
+  const { kernelStorage, hostStorage } = initSwingStore();
+  t.teardown(hostStorage.close);
```

Comment:
Would it make more sense for `initSwingStore` to take an argument that says we *do* want to open and return the `hostStorage`, rather than just opening it never referencing it, then closing it at the end of the test?

Reply:
It's not a question of opening or closing `hostStorage`. `close` is a host concern, so it's on the `hostStorage` facet. The test takes the place of the host, so it must get the host facet, and cleanup like a host would. BTW, cosmic-swingset doesn't cleanup either right now, maybe I should fix that in this PR as well?